src/components/Buildbaron/BuildBaronTab.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import { Tab } from "@leafygreen-ui/tabs";
+import BuildBaron from "components/Buildbaron/BuildBaron";
+import { BuildBaronQuery, BuildBaronQueryVariables } from "gql/generated/types";
+import { GET_BUILD_BARON } from "gql/queries";
+
+interface Props {
+  taskId: string;
+  execution: number;
+  setshowbuildbarontab: React.Dispatch<React.SetStateAction<Boolean>>;
+}
+
+export const BuildBaronTab: React.FC<Props> = ({
+  taskId,
+  execution,
+  setshowbuildbarontab,
+}) => {
+  const {
+    data: buildBaronData,
+    error: buildBaronError,
+    loading: buildBaronLoading,
+  } = useQuery<BuildBaronQuery, BuildBaronQueryVariables>(GET_BUILD_BARON, {
```

Comment:
the execution number can be changed from the task page from the execution select dropdown
![Screen Shot 2020-09-29 at 12 20 38 PM](https://user-images.githubusercontent.com/10734386/94585590-397b0400-024e-11eb-95c0-65c874c98a05.png)
We should refetch build baron data within a `useEffect` when the execution number changes